substrate-examples/offchain-worker/README.md

```diff
@@ -0,0 +1,70 @@
+# Proof of Concept: Tag Analysis on Origin
+
+This is a copy of [substrate's pallet template](https://github.com/substrate-developer-hub/substrate-node-template/tree/e0c480c0f322d0b0d1b310c93fa646fc0cfdd2df/pallets/template), enriched with a MIRAI [tag-analysis](https://github.com/facebookexperimental/MIRAI/blob/main/documentation/TagAnalysis.md). It is a first proof-of-concept for a [validation of unsigned transactions](https://github.com/bhargavbh/MIRAI/blob/main/substrate_examples/unsigned-transaction/description.md).
+
+# Running
+
+To run the analysis [install mirai](https://github.com/facebookexperimental/MIRAI/blob/main/documentation/InstallationGuide.md) and run
+
+`cargo mirai`
+
+from within this folder. The [config.toml](.cargo/config.toml) makes sure, that the analysis only runs on the function [`mirai_check.code_to_analyze`](src/mirai.rs).
```

Comment:
link to config.toml doesn't work. see also comment above

Reply:
name of function is wrong, too

Reply:
I changed the function name to be the same as in the `pallet_template` example. Then it is the same in both.